HW04/Vladislav_Mitin/lib/mentor.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Mentor
+  attr_reader :name, :surname, :notifications, :subscriptions
```

Comment:
Don't use public attr_reader if you don't need this method outside the class
```suggestion
  attr_reader :name, :surname, :notifications
  # ...
  private 
  
  attr_reader :subscriptions
```